Backend/app/provider-platform/dynamic-offer-driver-drainer/src/Types/DBSync/Create.hs

```diff
@@ -21,15 +21,17 @@ data DBCreateObject = DBCreateObject
 
 instance FromJSON DBCreateObject where
   parseJSON = A.withObject "DBCreateObject" $ \o -> do
-    A.Array a <- o A..: "contents" -- TODO remove "contents" field after roll out
-    contentsObj <- flip (A.withObject "last contents") (V.last a) $ \obj -> do
-      -- can we use contents_v2?
-      obj A..: "contents"
     contentsV2 <- o A..: "contents_v2"
     command <- contentsV2 A..: "command"
     tagObject :: DBModelObject <- command A..: "tag"
     contents <- command A..: "contents"
     mbMappings <- o A..:? "mappings"
+    contentsObj <-
+      o A..:? "modelObject" >>= \case
+        Just (A.Object obj) -> pure obj
+        _ -> do
```

Comment:
if euler part already deployed https://github.com/nammayatri/euler-hs/pull/8/files then this case will never happen 
Anyway can be removed later

Reply:
it's not deployed..we have to deploy drainer first then deploy the euler code as that change will be not compatible with old drainer.